src/workspaces/features/RunPipelineDialog/RunPipelineDialog.tsx

```diff
@@ -33,8 +35,33 @@ type RunPipelineDialogProps = {
 const RunPipelineDialog = (props: RunPipelineDialogProps) => {
   const router = useRouter();
   const { open, onClose, pipeline } = props;
+  const [showVersionPicker, setShowVersionPicker] = useState(false);
   const clearCache = useCacheKey(["pipelines", pipeline.code]);
 
+  const [fetch, { data, loading }] = useLazyQuery<PipelineCurrentVersionQuery>(
+    gql`
+      ${ParameterField.fragments.parameter}
+      query PipelineCurrentVersion(
+        $workspaceSlug: String!
+        $pipelineCode: String!
+      ) {
+        pipelineByCode(workspaceSlug: $workspaceSlug, code: $pipelineCode) {
+          currentVersion {
+            number
+            createdAt
+            user {
+              displayName
+            }
+            parameters {
+              ...ParameterField_parameter
+            }
+          }
+        }
+      }
+    `,
+    { fetchPolicy: "no-cache" }
+  );
+
   const form = useForm<{ version: PipelineVersion; [key: string]: any }>({
     async onSubmit(values) {
       const { version, ...params } = values;
```

Comment:
the version will always be provided now no ?

Reply:
Yes